app/components/grades/GradeDropdown.jsx

```diff
@@ -3,31 +3,19 @@ import _ from 'lodash';
 import {connect} from 'react-redux';
 import {withTranslation} from 'react-i18next';
 import {bindActionCreators} from 'redux';
-import {resolveLabel} from '../../utils/translationUtil';
+
+import {resolveLabel} from '../../utils';
 import {actionCreators} from '../../actions/index';
-import {sortData} from '../rooms/helpers/formHelpers';
-import {
-    getClassList,
-    getSubjectList,
-    createDropdown,
-    notifyIfEmpty,
-    getClassroomName
-} from '../helpers/dropdowns';
+import {sortData} from '../classroom/formHelpers';
+import {getClassList, getSubjectList, createDropdown, notifyIfEmpty, getClassroomName} from '../helpers/dropdowns';
 import css from './styles/grades.css';
 
-const GradeDropdown = ({t, classData, gradeData, subjectData, actions}) => {
+function GradeDropdown({t, classData, gradeData, subjectData, actions}) {
     const {subDrop, subjectName, classroomId, classroomDropdown} = gradeData;
     const openIt = {subDrop};
     const classOptions = getClassList(sortData(classData));
-
-    const classroom = _.isNull(classroomId)
-        ? classroomId
-        : getClassroomName(classroomId, classData.classData);
-
-    const subjectOptions = getSubjectList(
-        {selectedRoom: classroomId},
-        subjectData
-    );
+    const classroom = _.isNull(classroomId) ? classroomId : getClassroomName(classroomId, classData.classData);
```

Comment:
@hcuffy I don't know if this is an issue or if this is intentional:

if `classrootmId` is null, you want to use it?